packages/components/src/animate/README.md

```diff
@@ -0,0 +1,39 @@
+# Animate
+
+Simple interface to introduce animations to components. 
+
+## Usage
+
+```jsx
+import { Animate } from '@wordpress/components';
+
+const MyAnimatedNotice = () => (
+	<Animate todo="Add missing props">
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Animation finished.</p>
+			</Notice>
+		) }
+	</Animate>
+);
+```
+
+## Props
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`type` | `string` | `undefined` | Type of the animation to use.
+`options` | `object` | `{}` | Options of the chosen animation.
+`children` | `function` | `undefined` | A callback receiving a list of props ( `className` ) to apply to the DOM element to animate.
+
+## Available Animation Types
+
+### appear
+
+This animation is meant for popover/modal content, such as menus appearing. It shows the height and width of the animated element scaling from 0 to full size, from its point of origin. 
+
+#### Options
```

Comment:
Is there a reason that `options` ought to be a separate object, vs. just having its own properties be props of the component itself?

In other words, is...

```jsx
<Popover type="appear" options={ { origin: 'top' } } />
```

...necessarily any better than:

```jsx
<Popover type="appear" origin="top" />
```

?

Reply:
My thinking at the time is that "options" are type-specific, their format change for each animation type.

Reply:
> My thinking at the time is that "options" are type-specific, their format change for each animation type.

Okay, thanks for clarifying. I'm not sure it would necessarily be _wrong_ to still flatten these as top-level props, even if each `type` had their own handling and own set of options, but the rationale you give is sensible.

It may have been misleading to me because in the current implementation, the only option is `origin`, and it is consistent for both the `appear` and `slide-in` types, though I suppose that's more a reflection of its current state, and these may be expanded at some point in the future.